package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "engineering-playbook",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start --port 4000",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve --port 4000",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids"
+  },
+  "dependencies": {
+    "@cmfcmf/docusaurus-search-local": "^0.6.6",
+    "@docusaurus/core": "2.0.0-beta.5",
```

Is there a specific reason that we're not targeting a more recent update to Docusarus? I would recommend we update and pin to the latest Beta version at the time of this comment.

```suggestion
    "@docusaurus/core": "2.0.0-beta.16",
```

---

oh, good suggestion! I'll check the other dependencies for updates too just in case

---

ooh, already noticing nice new features 😄

---

🤔  hmm, I've never worked with renovatebot, but I'll take a look at it today and see if that makes sense

---

I've worked with it before so I might be able to help. Ping me on slack and we could pair on it.

---

I did it: #221

---

I've started playing around with this on other repos! way better than dependabot imo